### PR TITLE
add css and scripts config example

### PR DESCRIPTION
**Pull request by NicolasMassart**
# Pull request checklist

Use the following template to make sure your PR fits the Doctools documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-locally).

## Describe the change

Add config for extra CSS and scripts as a comment template in `mkdocs.extra.yml`

## Issue fixed

Documents ConsenSys/doctools.action-builder#6

## Impacted parts

For content changes:

- [x] Documentation content
- [x] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (e.g., lint or vale)
- [ ] MkDocs templates
- [x] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [x] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

File: docs/create/advanced/add-css-and-js.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Add custom CSS and Javascript
+
+The following guidelines explain how to add CSS and Javascript to your site without modifying
+the common doc theme.
+
+Extra script and style configuration enables individual sites to have a specific colour scheme
+or run some script to complement the initial set provided by the common theme.
+
+These scripts and styles will be loaded on all pages of the doc site.
```
Comment (alexandratran) [inline docs/create/advanced/add-css-and-js.md:13]:
```suggestion
# Add custom CSS and JavaScript

The following guidelines explain how to add CSS and JavaScript (JS) to documentation sites that use the [new system](../../overview/index.md#new-documentation-system) without modifying the common documentation theme.

Extra script and style configuration enables individual sites to have a specific color scheme
or to run scripts in addition to the common set of scripts.

These scripts and styles are loaded on all pages of the documentation site.
```

File: docs/create/advanced/add-css-and-js.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Add scripts and styles by updating the `mkdocs.extra.yml` configuration file.
+
+!!! example "Custom CSS and JS files example"
+
+    Create new `docs/assets/stylesheets/` and `docs/assets/javascripts/` folders.
+
+    Create a CSS files `docs/assets/stylesheets/a_custom_style.css` and a javascript file  `docs/assets/javascripts/a_custom_script.js`.
+
+    Modify the `mkdocs.extra.yml` with the following:
```
Comment (alexandratran) [inline docs/create/advanced/add-css-and-js.md:23]:
```suggestion
    Create a CSS file `docs/assets/stylesheets/a_custom_style.css` and a JS file  `docs/assets/javascripts/a_custom_script.js`.

    Modify `mkdocs.extra.yml` with the following:
```

File: mkdocs.navigation.yml [hunk 3]
```diff
@@ -36,6 +36,7 @@ nav:
       - Create a new documentation site: create/create-doc-site.md
       - Advanced:
           - Update or create a site template: create/advanced/create-site-template.md
+          - Add custom CSS and Javascript: create/advanced/add-css-and-js.md
```
Comment (alexandratran) [inline mkdocs.navigation.yml:39]:
```suggestion
          - Add custom CSS and JavaScript: create/advanced/add-css-and-js.md
```

**Review by alexandratran — COMMENTED** [review]
Some suggestions

File: docs/create/advanced/add-css-and-js.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+description: How to add CSS and Javascript
+---
+
+# Add custom CSS and Javascript
+
+The following guidelines explain how to add CSS and Javascript to your site without modifying
```
Comment (bgravenorst) [inline docs/create/advanced/add-css-and-js.md:7]:
```suggestion
The following guidelines explain how to add CSS and JavaScript to your site without modifying
```

File: docs/create/advanced/add-css-and-js.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+These scripts and styles will be loaded on all pages of the doc site.
+
+Add scripts and styles by updating the `mkdocs.extra.yml` configuration file.
+
+!!! example "Custom CSS and JS files example"
+
+    Create new `docs/assets/stylesheets/` and `docs/assets/javascripts/` folders.
+
+    Create a CSS files `docs/assets/stylesheets/a_custom_style.css` and a javascript file  `docs/assets/javascripts/a_custom_script.js`.
```
Comment (bgravenorst) [inline docs/create/advanced/add-css-and-js.md:21]:
```suggestion
    Create a CSS files `docs/assets/stylesheets/a_custom_style.css` and a JavaScript file  `docs/assets/javascripts/a_custom_script.js`.
```

File: mkdocs.extra.yml [hunk 6]
```diff
@@ -36,3 +36,9 @@ extra:
       link: https://consensys.net/quorum/developers
   version:
     switch: true # displays the version dropdown
+
+# Uncomment and add path if needed to have custom extra scripts and CSS
```
Comment (bgravenorst) [inline mkdocs.extra.yml:40]:
```suggestion
# Uncomment and update the path if you're using custom scripts and CSS
```

File: mkdocs.navigation.yml (hunk 3 above)
Comment (bgravenorst) [inline mkdocs.navigation.yml:39]:
```suggestion
          - Add custom CSS and JavaScript: create/advanced/add-css-and-js.md
```

**Review by bgravenorst — APPROVED** [review]
Minor nitpicks, LGTM

**Pull request merged**